src/components/PRList.tsx

```
@@ -155,13 +175,15 @@ function PRListContent({ searchQuery, onRefresh }: PRListContentProps) {
             </div>
 
             <div className="form-control self-end">
-              <button onClick={onRefresh} className="btn btn-primary">
-                Atualizar Dados
+              <button onClick={onRefresh} className="btn btn-primary gap-2">
+                <RefreshCw size={18} />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)